migrations/privateProgramInProjects/deleteAndUpdateAttachmentsInProject.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * name : updatePrivateProgramInProject.js
```

Comment:
@ankitpws the name is not as per the file.

Folders will be - updateProjectDocuments/removeAttachmentsNotUploadedInCloud.js

Folders will be - updateProjectDocuments/migratePrivateProjectToPublicProgram.js